### PR TITLE
Allow validateIn for type array

### PR DESCRIPTION
**Pull request by dontbug**
Allow for validation:

``` php
$inputs = [
    'values' => ['one']
];
$rules = [
    'values' => 'required|in:one,two'
];
$v = Validator::make($inputs, $rules);
var_dump(!$v->fails());
```

### CODE REVIEW

File: src/Illuminate/Validation/Validator.php [hunk 1]
```diff
@@ -962,7 +962,10 @@ protected function getSize($attribute, $value)
      */
     protected function validateIn($attribute, $value, $parameters)
     {
-        return in_array((string) $value, $parameters);
+        if(is_array($value))
+            return (count(array_diff($value, $parameters)) == 0);
+        else
+            return in_array((string) $value, $parameters);
```
Comment (franzliedke) [inline src/Illuminate/Validation/Validator.php]: Please don't leave out the braces. It should be like this:

~~~php
if (is_array($value)) {
    return count(array_diff($value, $parameters)) == 0;
} else {
    return in_array((string) $value, $parameters);
}
~~~

Comment (Talean-dev) [inline src/Illuminate/Validation/Validator.php]: ok. sorry. thanks

File: src/Illuminate/Validation/Validator.php [hunk 2]
```diff
@@ -962,7 +962,10 @@ protected function getSize($attribute, $value)
      */
     protected function validateIn($attribute, $value, $parameters)
     {
-        return in_array((string) $value, $parameters);
+        if(is_array($value))
```
Comment (GrahamCampbell) [inline src/Illuminate/Validation/Validator.php]: You must include braces. Follow PSR-2.

**Comment by GrahamCampbell** [comment]
Please fix the cs and send to 5.1 as one commit.

**Pull request closed without merging**